scout/server/extensions/chanjo2_extension.py

```diff
@@ -66,13 +66,21 @@ def get_gene_complete_coverage(
             "interval_type": "genes",
             "samples": [],
         }
+        analysis_types = []
+
         for ind in individuals:
             if not ind.get("d4_file"):
                 continue
 
             gene_cov_query["samples"].append(
                 {"coverage_file_path": ind["d4_file"], "name": ind["individual_id"]}
             )
+            analysis_types.append(ind.get("analysis_type"))
+
+        if "wes" in analysis_types:
```

Comment:
This reflect what happens for the other types of chanjo2 reports --> https://github.com/Clinical-Genomics/scout/blob/cbc426a3324cc9434c315b1f6b5a4753da97cd71/scout/server/blueprints/cases/templates/cases/chanjo2_form.html#L26

But it makes sense to me to make it use the same analysis type of the affected, as you suggest. I don't know, what do you think?

Reply:
I think it is fine on some level to leave it like this; there is a certain logic to it as well. But I do think from the analysts perspective, the affecteds are what drives the question.

Reply:
We can always change it later!